awx_collection/test/awx/conftest.py

```diff
@@ -17,6 +17,7 @@
 
 from ansible.module_utils.six import raise_from
 
+
```

Comment:
let's remove these new lines unless the linter is wanting them there

Reply:
Good call, thank you! That has been resolved in this [commit](https://github.com/ansible/awx/pull/16061/commits/d3d42ff2d48ba8598e90e8c5de1613fba769ff91)